### DISCUSSION TITLE
openapi/midware/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1344] [Announcements]
# openapi/midware/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/openapi/midware/

**Comment by hxx666** [comment]
为什么我生成用生成的密钥访问https://demo-dev.daocloud.io/apis/ghippo.io/v1alpha1/clusters接口时返回了200,但是items是空,实际是有集群的,但是我在用浏览器中密钥访问同样的接口items是有数据返回的
